docs/kubernetes/windows_demo.md

```diff
@@ -0,0 +1,181 @@
+# Kubernetes Basic Windows User Guide
+This guide gives an example on how to use SMB share(s) in pods running on Windows.
+
+## Prerequisites
+- Setup a Windows cluster following [this guide](https://github.com/kubernetes/kubernetes/blob/master/cluster/gce/windows/README-GCE-Windows-kube-up.md).
+- A non-cluster Windows 1809 VM with `Full` access to **Storage** under `Cloud API Access Scopes`.
```

Comment:
Link this to: https://cloud.google.com/compute/docs/access/create-enable-service-accounts-for-instances#changeserviceaccountandscopes